velox/expression/tests/FuzzerRunner.h

```diff
@@ -43,7 +42,8 @@
 ///
 /// The important flags that control Fuzzer's behavior are:
 ///
-///  --steps: how many iterations to run
+///  --steps: how many iterations to run.
+///  --duration_sec: alternatively, for how many seconds it should run.
```

Comment:
nit: Could we also tell users which one takes effect if both steps and duration_sec are specified?